internal/pkg/aws/route53/errors.go

```diff
@@ -3,7 +3,19 @@
 
 package route53
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+)
 
 // ErrNoDefaultCluster occurs when the default cluster is not found.
 var ErrDomainNotExist = errors.New("domain does not exist")
```

Comment:
What's the difference with this error? Can we delete this one now? (also its comment is outdated 🙈 )

Reply:
This error is used when the domain hosted zone isn't found. Previously we used this to identify whether the domain lives in the account, because when a domain is registered, the domain hosted zone is automatically created.

Maybe I should change this to "domain hosted zone not found"?

Reply:
ah okay that makes sense! yea I think your suggestion sounds good to me :)